interface/modules/custom_modules/oe-module-prior-authorizations/public/index.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ *  package OpenEMR
+ *  link    https://www.open-emr.org
+ *  author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  Copyright (c) 2022.
+ *  All Rights Reserved
+ */
+
+require_once dirname(__FILE__, 5) . "/globals.php";
+require_once dirname(__FILE__, 2) . '/vendor/autoload.php';
```

Comment:
if above autoload.php needed here?

Reply:
disregard above. looks like need to show your custom classed. But then i think it will bring in outdated eventlisteners classes too.

Reply:
Yes, that is how the module is built on auto-loading classes from its own namespace.
The namespace can be refactored to Juggernaut\PriorAuthModule.